ui/nuxeo-routing-behavior.js

```diff
@@ -71,22 +71,29 @@ export const RoutingBehavior = {
   _computeUrlFor() {
     return function(...args) {
       if (this.router) {
-        const route = args[0];
         const baseUrl = this.router.baseUrl || '';
+        let [route, ...params] = args;
+        if (!route) {
+          return '';
+        }
+        if (typeof route === 'object') {
```

Comment:
would rather rely on `route['entity-type']` and call that route by convention (and of course adjusting the routes to take the object as parameter too) or for compat purposes do a switch per known entity type to call the proper url functions with the right parameters.

Reply:
But we do have situations where we want to call different named routes for the same entity type. For the document entity-type, we might want to call both `document` and the `browse` routes, according to whether we need to generate a URL by UID or path.